float64 x
float64 y
float64 z
float64 roll
float64 pitch
float64 yaw
---
float64 resu1
float64 resu2
float64 resu3
float64 resu4
float64 resu5
float64 resu6
